modules/registry-operator-default-crd.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-default-crd-{context}"]
+= Image Registry Operator Custom Resource Definition and parameter values
+
+In {product-title}, the `Registry` Operator controls the registry feature. The
+Operator is defined by the `config.imageregistry.operator.openshift.io` Custom
+Resource Definition (CRD).
+
+////
+The following CRD displays the default configuration
+and explains both the parameters you can configure and valid parameter values:
+
+
+.Default <resource> CRD
```

Comment:
@dmage - another question: Is there a default `config.imageregistry.operator.openshift.io` CRD?

Reply:
@bmcelvee yes, the image registry operator (in managed state) always creates configs.imageregistry.operator.openshift.io/cluster when it doesn't exist.